widgetsnbextension/package.json

```diff
@@ -7,6 +7,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack",
+    "update:widgets": "rimraf node_modules/jupyter-js-widgets; npm install file:../jupyter-js-widgets; npm run build",
```

Comment:
This is *so* frustrating that we have to nuke a folder to make this work... (just commiserating)